docs/form-editor.md

```diff
@@ -0,0 +1,21 @@
+# Using the Form Editor
+
+## **Using the ConfigMap Properties Form**
```

Comment:
This section needs to be rewritten to be more general - since we now have Form editors for a number of resource types, not just ConfigMaps.

Reply:
Hi @olensmar, I will write this doc with the review help of @devcatalin.